### PR TITLE
Remove red login border on firefox

### PR DESCRIPTION
**Pull request by ManfredRichthofen**
Keeps the required input text box while removing the red border

Changes
Removes the red border you see when logging in on Firefox
**Before**
before
**After**
after

### CODE REVIEW

**Comment by KeepBotting** [comment]
support

File: src/elements/emby-input/emby-input.css [hunk 1]
```diff
@@ -21,6 +21,10 @@
         border: 0;
     }
 
+input:required { 
+box-shadow:none;
```
Comment (MrTimscampi) [inline src/elements/emby-input/emby-input.css]: Please use the following as a selector, to be more precise in your change:

`.emby-input:required`

Please also respect the formatting of the surrounding code (indentation and spacing)

File: src/elements/emby-input/emby-input.css [hunk 2]
```diff
@@ -17,10 +17,14 @@
     width: 100%;
 }
 
-    .emby-input::-moz-focus-inner {
-        border: 0;
+.emby-input::-moz-focus-inner {
+    border: 0;
     }
```
Comment (dkanada) [inline src/elements/emby-input/emby-input.css]:
```suggestion
}
```

File: src/elements/emby-input/emby-input.css [hunk 3]
```diff
@@ -17,9 +17,13 @@
...
 
-    .emby-input::-moz-focus-inner {
-        border: 0;
-    }
+.emby-input::-moz-focus-inner {
+    border: 0;
+}
+
+.emby-input:required { 
+    box-shadow:none;
```
Comment (MrTimscampi) [inline src/elements/emby-input/emby-input.css]:
```suggestion
    box-shadow: none; 
```

**Pull request merged**